src/core/operations/RegularExpression.mjs

```diff
@@ -207,7 +207,7 @@ function regexList(input, regex, displayTotal, matches, captureGroups) {
         if (captureGroups) {
             for (let i = 1; i < match.length; i++) {
                 if (matches) {
-                    output += "  Group " + i + ": ";
+                    output += "  Group " + i + " (" + regex.xregexp.captureNames[i - 1] + "): ";
```

Comment:
There needs to be a check to ensure that the current group is named.  Otherwise, in the case of no names there is the following error:

Invalid regex. Details: regex.xregexp.captureNames is null

And if there are some that are named and some that are not, the unnamed ones gain a label of 'null', which isn't ideal.